### ISSUE TITLE
dowload metadata error

### ISSUE THREAD

**Issue by sarpiens** [EBI-Metagenomics/emg-toolkit#30]
Hello,
I'm trying to use mg-toolkit (version 0.10.0) to fetch metadata from a large Project PRJEB11419. After hours of execution I get the following error:
![mg-toolkit_error](https://user-images.githubusercontent.com/49655743/171233252-2c7e9b39-f54d-4300-8037-f28aa6777f71.png)

I have tried with other projects, and I have only been able to reproduce this error with this particular dataset.
Thanks in advance for any help that you can provide!

**Comment by mberacochea** [comment]
Hi @sarpiens , thank you for reporting this problem. I suspect that problem is due to the ENA API having issues, I'll take a look and get back to you (next week as am I'm off this week :)).

**Comment by sarpiens** [comment]
Hi! There are any news on this issue?
Thanks again for any help that you can provide!

**Comment by mberacochea** [comment]
Hi @sarpiens , ENA issues were solved. I will run the download process for that project now, if I can reproduce the error then I should be able to fix it. I'll keep you posted.

**Comment by mberacochea** [comment]
Alright... that study is humongous, and this tool is not handling network issues properly. The best way to deal with this is to:
 - Improve the network error handling. Retrying an ENA http request if it fails and to avoid crashing the app if one sample fails, for example
 - Add a --resume or make the app read the csv output file to try to restart without re-fetching everything again.

We will improve this, but unfortunately I can't provide an ETA.

**Comment by sarpiens** [comment]
Okey thanks. So I guess this will be improve in future versions, right?

**Comment by mberacochea** [comment]
Yes, we will improve in future versions. I've created this ticket for this new feature https://github.com/EBI-Metagenomics/emg-toolkit/issues/31
